src/main/java/com/artipie/settings/repo/RepositoriesFromStorage.java

```diff
@@ -104,7 +104,7 @@ public CompletionStage<RepoConfig> config(final String name) {
                 content.key,
                 Yaml.createYamlInput(data).readYamlMapping(),
                 this.cache,
-                new MetricsContext(this.settings.meta()).enabled()
+                new MetricsContext(this.settings.meta()).storage()
```

Comment:
This class has to get `MetricsContext` from outside.
`MetricsContext` should be created only ones.